### PR TITLE
Sync from upstream repo 09/09/2024 21:56:08

### PR DESCRIPTION
**Pull request by momo-the-monster**
Changes Files:

### CODE REVIEW

File: Docs/docs/guides/faq.md [hunk 1]
```diff
@@ -3,10 +3,10 @@
 **V**RChat **C**reator **C**ompanion. It's the official tool for creating and updating Unity projects for VRChat. Read more about the Creator Companion on our [glossary page](https://vcc.docs.vrchat.com/guides/glossary/).
 
 ## Where are backups stored?
-When you click "Make Backup" in the VCC, a `.zip` file of your project will be saved in `%LocalAppData%/VRChatCreatorCompanion/Project Backups`. You can change this location in Settings, under the General tab - scroll down and look for "Backups".
+When you click "Make Backup" in the VCC, a `.zip` file of your project will be saved in `%LocalAppData%/VRChatCreatorCompanion/Logs/Project Backups`. You can change this location in Settings, under the General tab - scroll down and look for "Backups".
```
Comment (momo-the-monster) [inline Docs/docs/guides/faq.md:6]: Hm, this isn't right - guess it snuck into the main branch?

Comment (momo-the-monster) [reply Docs/docs/guides/faq.md:6]: Nope - the other way around. Fixed here but not in the main branch.

**Comment by megatron-DOTM** [comment]
HI

**Comment by megatron-DOTM** [comment]
I hope to have my changes reviewed and merged into your documentation.

**Pull request closed without merging**